db-scripts/src/main/resources/cgds.sql

```diff
@@ -420,6 +420,7 @@ CREATE TABLE `structural_variant` (
   `LENGTH` int(11),
   `COMMENTS` varchar(255),
   `EXTERNAL_ANNOTATION` varchar(80),
+  `IS_GERMLINE` tinyint(1) NOT NULL,
```

Comment:
We are also missing `SITE1_HUGO_SYMBOL` and `SITE2_HUGO_SYMBOL`.

Reply:
can the entrez fields perhaps be used?